telegram/ext/conversationhandler.py

```diff
@@ -294,6 +303,14 @@ def handle_update(self, update, dispatcher):
         """
         new_state = self.current_handler.handle_update(update, dispatcher)
 
+        if self.timeout_job is not None:
+            self.timeout_job.schedule_removal()
```

Comment:
Do I understand correctly that there is only a single timeout job which gets removed whenever any conversation is triggered? 

Wouldn't you want to use a different job for each conversation?

Reply:
This timeout job is triggered in any conversation state, if you mean this. For different ConversationHandler instances the job is different too.